packages/api/package.json

```
@@ -30,38 +30,38 @@
   },
   "cldr-data-coverage": "full",
   "devDependencies": {
-    "@babel/cli": "^7.12.1",
-    "@babel/core": "^7.12.3",
-    "@babel/preset-env": "^7.12.1",
-    "@babel/preset-react": "^7.12.5",
-    "@babel/preset-typescript": "^7.12.1",
-    "@types/node": "^14.14.6",
+    "@babel/cli": "^7.13.10",
+    "@babel/core": "^7.13.10",
+    "@babel/preset-env": "^7.13.10",
+    "@babel/preset-react": "^7.12.13",
+    "@babel/preset-typescript": "^7.13.0",
+    "@types/node": "^14.14.35",
     "@types/react": "^16.9.55",
     "babel-plugin-istanbul": "^6.0.0",
     "babel-plugin-transform-inline-environment-variables": "^0.4.3",
     "cldr-data": "^36.0.0",
-    "concurrently": "^5.3.0",
-    "core-js": "^3.6.5",
-    "eslint": "^7.12.1",
-    "eslint-plugin-prettier": "^3.1.4",
-    "eslint-plugin-react": "^7.21.5",
+    "concurrently": "^6.0.0",
+    "core-js": "^3.9.1",
+    "eslint": "^7.22.0",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "globalize-compiler": "^1.1.1",
     "iana-tz-data": "^2019.1.0",
-    "node-dev": "^6.2.0",
-    "prettier": "^2.1.2",
-    "typescript": "^4.0.5"
+    "node-dev": "^6.4.0",
+    "prettier": "^2.2.1",
+    "typescript": "^4.2.3"
   },
   "dependencies": {
     "botframework-webchat-core": "^0.0.0-0",
-    "event-target-shim": "5.0.1",
-    "event-target-shim-es5": "1.2.0",
-    "globalize": "1.6.0",
-    "math-random": "2.0.1",
-    "prop-types": "15.7.2",
+    "event-target-shim": "^5.0.1",
```

```suggestion
    "event-target-shim": "5.0.1",
```

For all production dependencies, we pinned them down.